docs/_includes/docs_contents_mobile.html

```diff
@@ -1,15 +1,15 @@
 <div class="docs-nav-mobile unit whole show-on-mobiles">
-  <select onchange="if (this.value) window.location.href=this.value">
+  <select id="doc-nav" onchange="navigateToUrl(this)" aria-label="Select a page from the documentation">
     <option value="">Navigate the docs…</option>
-    {% for section in site.data.docs_nav -%}
-    <optgroup label="{{ section.title }}">
-    {%- for item in section.docs -%}
-      {% assign p = site.docs | where: "url", item.link | first %}
-      <option value="{{ p.url | relative_url }}">
-        {{- p.menu_name | default: p.title -}}
-      </option>
-    {%- endfor %}
-    </optgroup>
-    {% endfor -%}
+    {% for section in site.data.docs_nav %}
+      <optgroup label="{{ section.title }}">
+        {%- for item in section.docs -%}
+          {% assign page = site.docs | where: "url", item.link | first %}
+          <option value="{{ page.url | relative_url }}">
+            {{- page.menu_name | default: page.title -}}
+          </option>
+        {%- endfor %}
+      </optgroup>
+    {% endfor %}
```

Comment:
I'm not following the reasoning for the re-indentation work in this PR. We've not indented the liquid in most pages because it's not present in the final site HTML, where as indenting it along with the HTML gives disjointed indentation in the final output.

Reply:
Hello @mattr-, the indentation has been adjusted solely to enhance readability in my editor.